packages/jsii-pacmak/lib/targets/go/util.ts

```diff
@@ -46,8 +46,39 @@ export function getFieldDependencies(fields: GoTypeMember[]): Package[] {
   }, []);
 }
 
+// export function getParameterDependencies(params: GoParameter[]): Package[] {
+//   return params.reduce((accum: Package[], param) => {
+//     return param.reference?.type?.pkg
+//       ? [...accum, param.reference?.type.pkg]
+//       : accum;
+//   }, []);
+// }
 const RESERVED_WORDS: { [word: string]: string } = {
+  break: 'break_',
+  default: 'default_',
+  func: 'func_',
+  interface: 'interface_',
+  select: 'select_',
+  case: 'case_',
+  defer: 'defer_',
+  go: 'go_',
```

Comment:
Note W/R/T how this map is used... It appears reserved words are slugified by appending an `_`. But this is unsafe in the event the slugified name is already present in the same scope...

Might want to get back to this in the future :)